dr-scripts_install.lic

```diff
@@ -0,0 +1,91 @@
+=begin
+
+	dr-scripts install helper
+	Aims to automate running through the steps from https://github.com/rpherbig/dr-scripts/wiki/First-Time-Setup
+	Please see us on Discord if there are any issues, or raise an issue on Github
+
+        author: Mahtra (mahtra@seul.org)
+        game: Dragonrealms
+        version: 1.0
+
+        changelog:
+                1.0 (2022-01-27):
+                        First release
+
+=end
+
+  def wait_for_script_to_complete(name, args = [], flags = {})
+    script_handle = start_script(name, args.map { |arg| arg =~ /\s/ ?  "\"#{arg}\"" : arg }, flags)
+    if script_handle
+      pause 2
+      pause 0.5 while Script.running.include?(script_handle)
+    end
+    script_handle
+  end
+
+  def download_dependency
+    if(File.exist?('./scripts/dependency.lic')) 
+      echo "Depdency exist already, skipping downloads"
+    else
+      echo "Downloading dependency"
+      wait_for_script_to_complete('repository', ['download', 'dependency.lic'])
+      pause 2
+    end
+  end
+  
+  def install_dependency
+    if(File.exist?('./scripts/common.lic')) 
```

Comment:
❓ Should this be checking if `./scripts/dependency.lic` exists?

Reply:
Actually, I might remove the check entirely. The premise was to check if dependency had already been run, and common.lic would only exist after dependency install... But, after discussion on Discord, I might skip the check and assume it hasn't.